CSS/Default.sublime-keymap

```diff
@@ -1,11 +1,24 @@
 [
+	// Add a colon and semicolon after a property name,
+	// if the next following token doesn't look like a value.
+	{ "keys": [":"], "command": "insert_snippet", "args": {"contents": ":$0;"}, "context":
```

Comment:
Personally, I find it questionable whether we want to match a colon with a semi-colon when `smart_typing` is disabled. We don't have a clear definition of what "smart typing" entails, but to me it includes anything that the user didn't request to insert explicitly (and that isn't covered by `auto_match_enabled`). For example, in this PR, a semi-colon is added after a property value completion iff `smart_typing` is active.

In general, I suppose we have to ask the question whether smart typing *levels* would be something we find worth implementing and whether it makes sense to differentiate between completing `:` with `:;` or `: ;`. And of course whether there are other instances of this that already exist. 
In my opinion, on and off are enough configuration by default. Any smart completion needs some getting used to and if more fine-tuned completions are desired, the user can tweak key bindings to their desire. It's not the easiest place for configuration, but at this level of granularity I think it is warranted.

Since `smart_typing` will be disabled by default, this is also something that would be backwards-incompatible if removed, as far as I understood, but we should consider whether it would be worth it.

Reply:
`auto_match_enabled` covers auto adding `begin` and `end` punctuation of sections such as strings or blocks. The value section of key-value pairs fits quite well into that definition, IMHO, if we don't intent to create typing helpers which auto add semicolons everywhere.

But you are right with `smart_typing` probably being too general to cover all possible use cases.

We might also make use of the default setting `"auto_complete_trailing_spaces": true,` to control space instead of introducing `smart_typing`.

Another question might be whether to add space in completions only, or whether to use the setting to gate key bindings as well. In former case we probably should remove adding spaces via bindings at all.